### PR TITLE
NO-JIRA: Minor wording tweaks for CLI help

### PR DESCRIPTION
**Pull request by bergerhoffer**
Minor updates to the CLI text for grammar and consistency (since this is pulled into product docs as the CLI reference docs).

For example, short names and example comments should have no ending punctuation, but long descriptions should.

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@arda PTAL

**Review by ardaguclu — COMMENTED** [review]
That looks great, thank you
/lgtm

File: pkg/cli/version/version.go [hunk 1]
```diff
@@ -37,10 +37,10 @@ var (
 		Pass --client to print only the OpenShift client version.
 	`)
 	versionExample = templates.Examples(`
-		# Print the OpenShift client, kube-apiserver, and openshift-apiserver version information for the current context
+		# Print the OpenShift client, Kubernetes API server, and OpenShift API server version information for the current context
```
[the remark this answers is not in the document]
Comment (bergerhoffer) [reply pkg/cli/version/version.go:40]: No worries, I can revert these ones

[the remark this answers is not in the document]
Comment (ardaguclu) [reply pkg/cli/version/version.go]: I think, we can merge this version of the PR and no need to make changes just for this

[the remark this answers is not in the document]
Comment (bergerhoffer) [reply pkg/cli/version/version.go]: Too late :)

**Comment by ardaguclu** [comment]
/retitle NO-JIRA: Minor wording tweaks for CLI help
/retest

**Pull request merged**